spec/controllers/catalog_controller_spec.rb

```diff
@@ -1268,4 +1268,31 @@
       end
     end
   end
+
+  describe '#common_st_record_vars' do
+    let(:st) { FactoryGirl.create(:service_template) }
+
+    before { controller.instance_variable_set(:@edit, :new => {:catalog_id => cat_id}) }
+
+    context 'Service Catalog Item without any Catalog' do
+      let(:cat_id) { "" }
```

Comment:
I'd rather put the whole 2nd argument into a `let(:edit)` so other people would understand more what's happening.